content/en/core/releases/3.14.0.md

```diff
@@ -0,0 +1,143 @@
+# 3.14.0 Release Notes
```

Comment:
This is the old markdown version of the file, not the new hugo version. Check out the [source code for the 3.13.0 release notes](https://raw.githubusercontent.com/medic/cht-docs/main/content/en/core/releases/3.13.0.md) to see what I mean. I'm guessing you used the release note generator script and it's probably out of date now that we're storing everything in cht-docs...

Reply:
:facepalm: I even had this in my head when I started, but yes, after using the release note generator script I forgot to come back and change this....  I will update the script so that it outputs the proper format.